pkg/codegen/docs/templates/index.tmpl

```diff
@@ -12,6 +12,11 @@ menu:
 {{ htmlSafe "<!-- Do not edit by hand unless you're certain you know what you are doing! -->" }}
 
 {{ htmlSafe .PackageDescription }}
+<!--
+    This comment ensures that there is always a new-line between the
+    package description and the start of the headers. This comment
+    will not be included in the generated markdown.
+-->
 
 {{- if ne (len .Modules) 0 -}}
```

Comment:
Hm. Does this work, though (as the `-` trims leading whitespace)?

```suggestion
{{ if ne (len .Modules) 0 -}}
```

Reply:
Yeah this does work. The latest commits in the docs PR were generated using this change.